deploy/sre-prometheus/100-important-routes-slo-PrometheusRule.yaml

```diff
@@ -0,0 +1,22 @@
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-important-routes-slo
+    role: alert-rules
+  name: sre-important-routes-slo
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-important-routes-slo-alerts
+    rules:
+    - alert: ConsoleRoute-SLO-Breach
+      annotations:
+        message: The console route has had 100 percent request failures for 60 seconds.
+      expr: |
+        (sum(rate(haproxy_backend_http_responses_total{route=~"console",code=~"(5..)"}[1m])) / sum(rate(haproxy_backend_http_responses_total{route=~"console",code=~"([1-5]..)"}[1m])) >= 1)
```

Comment:
why the extra parenthesis at the beginning?

Reply:
[Looks the same here](https://github.com/openshift/managed-cluster-config/blob/ecc37336cecb461e78cc111eea429fa53d5d4005/deploy/sre-prometheus/100-sre-api-errors.PrometheusRule.yaml#L17) so I thought to save the same syntax.

Reply:
can you make sure that's really needed and drop them if they're not please

Reply:
done :)